packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -200,6 +219,27 @@ export const prepareLocalOrchestrationAccountKit = (
             );
           }, 'Delegate');
         },
+        Deposit() {
+          trace('Deposit');
+          return zcf.makeInvitation(
+            seat => {
+              const { give } = seat.getProposal();
+              return watch(
+                zoeTools.localTransfer(
+                  seat,
+                  // @ts-expect-error LocalAccount vs LocalAccountMethods
+                  this.state.account,
+                  give,
+                ),
+                this.facets.seatExiterHandler,
+                seat,
+              );
+            },
+            'Deposit',
+            undefined,
+            M.splitRecord({ give: AnyNatAmountsRecord, want: {} }),
+          );
```

Comment:
Given `zoeTools.localTransfer` is expected to resolve promptly, we technically do not need to use `watch` here and instead could use `asVow`. 

This approach seems harmless, and appeases the linter. It also allows a developer to call `Deposit()` inside an async-flow if they wished.